frontend/src/main/pages/AdminJobsPage.js

```diff
@@ -71,16 +57,24 @@ const AdminJobsPage = () => {
                         <TestJobForm submitAction={submitTestJob} />
                     </Accordion.Body>
                 </Accordion.Item>
-                {
-                    jobLaunchers.map((jobLauncher, index) => (
-                        <Accordion.Item eventKey={index + 1}>
-                            <Accordion.Header>{jobLauncher.name}</Accordion.Header>
-                            <Accordion.Body>
-                                {jobLauncher.form}
-                            </Accordion.Body>
-                        </Accordion.Item>
-                    ))
-                }
+                <Accordion.Item eventKey="1">
+                   <Accordion.Header>Milk The Cows</Accordion.Header>
+                   <Accordion.Body>
+                       <JobComingSoon/>
+                   </Accordion.Body>
+               </Accordion.Item>
+               <Accordion.Item eventKey="2">
+                   <Accordion.Header>Instructor Report</Accordion.Header>
+                   <Accordion.Body>
+                       <InstructorReportJobForm submitAction={submitTestJob} />
+                   </Accordion.Body>
+               </Accordion.Item>
+               <Accordion.Item eventKey="3">
+                   <Accordion.Header>Update Cow Health</Accordion.Header>
+                   <Accordion.Body>
+                       <UpdateCowHealthJobForm submitAction={submitTestJob} />
```

Comment:
Again: this can't be `submitTestJob`.  You need a separate form for each of the different jobs.

And why are you leaving out the MilkTheCows job?   It needs the same treatment.  If it wasn't mentioned in the issue, that's only because the previous issue that this team did to put in a placeholder for the MilkTheCows job sort of assumed that you'd finish the job and replace that with a form to submit MilkTheCows once the backend was done.   So you might as well do that one here too.   It doesn't make sense to do two of them and leave the other undone.